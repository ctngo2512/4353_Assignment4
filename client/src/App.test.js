import React, {useEffect} from 'react';
import App from './App';
import Hero from './Hero';
import {handleLogout} from './App';
import {authListener} from './App';
import {configure, shallow, mount} from 'enzyme';
import {act, render} from '@testing-library/react';
import fire from './fire';

describe('App component tests', ()=> {
      
    it("renders without crashing", () => {
        shallow(<App />);
    });

    test('should handle login and hero', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
        const wrapper2 = mount(<Hero handleLogout = {handleLogout}/>);
        expect(wrapper2).toMatchSnapshot();
    });

});