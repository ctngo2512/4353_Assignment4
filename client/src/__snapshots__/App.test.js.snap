// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component tests should handle login and hero 1`] = `ShallowWrapper {}`;

exports[`App component tests should handle login and hero 2`] = `ReactWrapper {}`;
